pages/teams.html

```diff
@@ -52,7 +52,8 @@ <h4 class="title" id="core">Core • <a href="https://chat.godotengine.org/chann
 		</p>
 
 		<p>
-			<span class="teams-subteam-leader">Juan Linietsky (<a href="https://github.com/reduz">@reduz</a>)</span>,
+			<span class="teams-subteam-leader">Juan Linietsky (<a
+					href="https://github.com/reduz">@reduz</a>)</span>,
```

Comment:
Are those formatting changes really improving things?
I had tried to format it with one name per line.